tests/parser/features/decorators/test_payable.py

```diff
@@ -91,21 +91,6 @@ def bar() -> bool:
     return True
 
 @external
-def __default__():
```

Comment:
Why are these test cases removed?

Reply:
I removed while debugging, and omitted to add them back. Apologies!